Formula/signadot-cli.rb

```diff
@@ -5,27 +5,27 @@
 class SignadotCli < Formula
   desc "Command-line interface for Signadot"
   homepage "https://signadot.com"
-  version "0.3.1"
+  version "0.3.2"
 
   on_macos do
     if Hardware::CPU.intel?
-      url "https://github.com/signadot/cli/releases/download/v0.3.1/signadot-cli_darwin_amd64.tar.gz"
-      sha256 "8881f132c8c4dfbb567b8604a242492510e003c4facfe41210c1857a479d1b09"
+      url "https://github.com/signadot/cli/releases/download/v0.3.2/signadot-cli_darwin_amd64.tar.gz"
+      sha256 "92d0ffe267e918a41fd73c677225aee3157766d8b92ace0334ee801f0d5099a4"
```

Comment:
What's a good way to verify the sha256 values? I downloaded the files and checked for their `sha256sum`, but that's giving me a different value than is populated here. Also, I'm getting the same sha256 values for all 4 files. I'm pretty sure the issue is on my side, as I see the same behavior with one of the previous update to this file.

<img width="1083" alt="image" src="https://user-images.githubusercontent.com/3791487/190009233-e7c91556-f0a5-4f42-a6c0-bbd4ee45a54b.png">

Reply:
Looks like you get the same sum b/c the files are all Not Found, which in turn is because the GitHub PAT I used in the release didn't work.

In the past I didn't verify the sums but rather the brew install with the resulting formula file, which probably checks the sums as well..

Looking into how to re-release with a good GitHub PAT.

Thanks for checking!

Reply:
Ah, I didn't realize that. But glad it led to something :)

The other one I checked against to verify sha computation was for the change ([link](https://github.com/signadot/homebrew-tap/commit/1bc3b6328c3e64965f9d75b163957aae6f11ecc5)) made by Anirudh. I'm getting the same sha256 sum for all four files which is different than the ones mentioned in the file. The files do exist in that case, so I must be doing something wrong myself as well during sha256 computation.

<img width="895" alt="image" src="https://user-images.githubusercontent.com/3791487/190011685-0a0fbdcb-09d7-4ee2-9408-835bd61456ff.png">

Reply:
Never mind about the last post. I think I'm doing curl incorrectly as the downloaded file had below content.

<img width="583" alt="image" src="https://user-images.githubusercontent.com/3791487/190012634-72ea0b90-b0db-46f0-b28a-f81300050596.png">

Reply:
ack, thx again for checking.  I've re-released with a new PAT, which led to new checksums, updated in this PR.

Reply:
I don't see any update to the checksums now. Can you check if you've committed changes correctly?

<img width="1747" alt="image" src="https://user-images.githubusercontent.com/3791487/190014513-68f0fbf3-5515-4b13-8388-b13b79c31ec7.png">

Reply:
I don't understand the diff, might be a GH bug, but the file contents contain the correct checksums which are different than the main branch.

BTW, wget works for downloading the files, there was a 302 redirect which seemed to be confusing curl.

Reply:
oh it was merged by @foxish but it still had the bottled line which is fixed in this PR.

you can check git log on main.

Reply:
cc @foxish -- brew install won't work:
```
22-09-13 scott@pavillion homebrew-tap % brew install signadot-cli
Error: signadot-cli: wrong number of arguments (given 1, expected 0)
```

There is the bottled line generated by GoReleaser which breaks it.  This PR removes that line and should work.